app/views/stats_graph.php

```diff
@@ -0,0 +1,41 @@
+<div id="graphdiv"></div>
+
+<script src="https://l10n.mozilla-community.org/~pascalc/web_l10n_stats/dygraph-combined.js"></script>
+
+<script>
+graph = new Dygraph(
+    document.getElementById("graphdiv"),
+    "https://l10n.mozilla-community.org/~pascalc/web_l10n_stats/logs/<?=$locale?>.csv", // path to CSV file
```

Comment:
Since we're inside a PHP file, does it make sense to cache this request locally?